index.html

```diff
@@ -1,71 +1,95 @@
-    <header class="main-nav">
-      <div class="brand-logo">
-        <a href="#">
-            <img class="logo-spin" src="assets/name-circle-r.svg" alt="Spinning Logo">
-            <img class="logo-still" src="assets/SMILE0.svg" alt="Still Logo">
-        </a>
-    </div>    
-      <nav>
-        <!-- Navigation menu with links to different sections of the page -->
-        <div class="nav-menu">
-          <button type="button" class="menu-icon"><img src="assets/icon-hamburger.svg" alt="Menu icon"></button>
-          <ul class="nav-links">
-            <li><a>Portfolio</a></li>
-            <li><a>About</a></li>
-            <li><a>Contact</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
-    <!-- Headline Section -->
-    <section class="intro-section">
-      <h1 class="headline">I’m Cristian.<br>Glad to see you!</h1>
-      <p class="subtext">I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-      <a class="connect-btn">LET'S CONNECT</a>
-      <ul class="social-icons">
-        <li><a href=""><img src="assets/twitter.svg" alt="Cristian's Twitter" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/linkedin.svg" alt="Cristian's LinkedIn" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/github.svg" alt="Cristian's GitHub" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/angel.svg" alt="Cristian's Angelist" class="social-icon"></a></li>
-      </ul>
-    </section>
-<!-- Projects Section -->
-<section class="project-section" aria-label="Projects">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+  <title>Personal Portfolio Website</title>
+</head>
+
+<body>
+  <!-- Main navigation bar or toolbar -->
+  <header class="main-nav">
+    <div class="brand-logo">
+      <a href="#about-section">
+        <img class="logo-spin" src="assets/name-circle-r.svg" alt="Spinning Logo">
+        <img class="logo-still" src="assets/SMILE0.svg" alt="Still Logo">
+      </a>
+    </div>
+    <nav>
+      <!-- Navigation menu with links to different sections of the page -->
+      <div class="nav-menu">
+        <button type="button" class="menu-icon" id="hamburger-icon"><img src="assets/icon-hamburger.svg"
+            alt="Menu icon"></button>
+        <ul class="nav-links">
+          <li><a href="#project-section">Portfolio</a></li>
+          <li><a href="#about-section">About</a></li>
+          <li><a href="#Contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] kindly consider making sure that your nav links are visible on the mobile menu cause currently it's empty to meet the project requirement and in order to increase its interactiveness<img width="377" alt="Screen Shot 2023-06-27 at 09 51 45" src="https://github.com/CVILLA09/Portfolio/assets/60146030/7b5f4c27-7c28-42c6-be22-1f6493d360d8"> 

while fixing that put into consideration fixing all requested [here](https://github.com/CVILLA09/Portfolio/pull/12#discussion_r1242895309) as requested by the previous reviewer